streamwebs_frontend/streamwebs/views.py

```diff
@@ -102,3 +119,127 @@ def user_login(request):
 def user_logout(request):
     logout(request)
     return HttpResponseRedirect('/streamwebs/')
+
+
+def water_quality(request, site_slug, data_id):
+    quality_data = {
+        'id': 1,
+        'date': '2016-06-10',
+        'site': {
+            'name': 'Test 1',
+            'slug': 'test1',
+            'site_type': 'steward',
+            'location': {'x': -122.31211423, 'y': 45.11231324}
+        },
+        'school': 'Education Middle School',
+        'DEQ_wq_level': 'B',
+        'fish_present': True,
+        'live_fish': 5,
+        'dead_fish': 2,
+        'air_temp_unit': 'F',
+        'water_temp_unit': 'C',
+        'sample_1': {
+            'water_temperature': 45,
+            'water_temp_tool': 'Vernier',
+            'air_temp': 52.3456,
+            'air_temp_tool': 'Manual',
+            'dissolved_oxygen': 55.2,
+            'oxygen_tool': 'Manual',
+            'pH': 6.2,
+            'pH_tool': 'Vernier',
+            'turbidity': 12.4,
+            'turbid_tool': 'Vernier',
+            'salinity': 5.8,
+            'salt_tool': 'Vernier',
+            'conductivity': 7.644,
+            'total_solids': .221,
+            'bod': 15.2,
+            'ammonia': 5.256,
+            'nitrite': 15.2,
+            'nitrate': 1.23,
+            'phosphates': 8.26,
+            'fecal_coliform': 54.2
+        },
+        'sample_2': {
+            'water_temp': 23,
```

Comment:
I think you mean 'water_temperature' for samples 2, 3, and 4 instead of 'water_temp'.